README.md

```diff
@@ -184,10 +184,16 @@ You can configure a Dict for each prefix so that the display is more readable.
 To make the guide pop up **Register the description dictionary for the prefix first**. Assuming `Space` is your leader key and the Dict for configuring `Space` is `g:which_key_map`:
 
 ```vim
-call which_key#register('<Space>', "g:which_key_map")
-
 nnoremap <silent> <leader> :<c-u>WhichKey '<Space>'<CR>
 vnoremap <silent> <leader> :<c-u>WhichKeyVisual '<Space>'<CR>
+
+call which_key#register('<Space>', "g:which_key_map")
+```
+
+The above registers the same description dictionary for both normal and visual modes. To use a different description dictionary for visual mode: add a third argument specifying which mode and use a different dictionary for visual mode:
+```vim
+call which_key#register('<Space>', "g:which_key_map", 0)
```

Comment:
Can be more descriptive if we use the mode as the optional argument, something like `call which_key#register('<Space>', "g:which_key_map", 'n')` and `call which_key#register('<Space>', "g:which_key_map", 'v')`.

Reply:
Done. I was following what which_key#start(vis, ...) did with 0,1.

I also updated more on the documentation, `doc/vim-which-key.txt`, and removed a reference to #155 in the `README.md` as it is no longer a problem, and that claimed fix with '<VisualBindings>' was nonsense anyway.